README.md

```diff
@@ -48,15 +56,24 @@ Huber, Anders Mörtberg.
 
 This makes it possible to directly represent higher inductive types.
 
-Maintainers
------------
 
-* [Anders Mörtberg](https://staff.math.su.se/anders.mortberg/)
+Reviewing of pull requests
+--------------------------
+If you switch your draft pull request (PR) to 'ready to merge',
+or directly create an open PR,
+we should request a review, by one of the reviewers below.
+If that doesn't happen, you can also request a reviewer yourself (for reviewer expertise see below),
+to make us aware of the open PR.
```

Comment:
```suggestion
to make us aware of the open PR. Feel free to use Discord to get in touch with a reviewer in case reviewing is taking a very long time.
```

Reply:
Ah - that's good!